code/game/machinery/vending.dm

```diff
@@ -1561,6 +1561,68 @@
 	idle_power_usage = 211
 	vendor_department = DEPARTMENT_OFFSHIP
 
+/obj/machinery/vending/style
+	name = "Style vendor"
```

Comment:
Get a better name.

Reply:
Dress-O-Matic

Reply:
Asters Guild Style-o-matic

anyone who disagrees is a pee pee poo poo head